sbin/pfmon

```diff
@@ -410,24 +410,22 @@ sub _runtask {
             next;
```

Comment:
I suggested to reset it to avoid a leftover time_taken value to be used in a calculation after someone disables and then enables a maintenance task.
Which could happen if the interval is changed to zero during the running of a task.